fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -227,38 +248,31 @@ Future<Void> deleteTenant(Reference<DB> db, TenantName name) {
 }
 
 ACTOR template <class Transaction>
-Future<std::map<TenantName, TenantMapEntry>> listTenantsTransaction(Transaction tr,
-                                                                    TenantNameRef begin,
-                                                                    TenantNameRef end,
-                                                                    int limit) {
-	state KeyRange range = KeyRangeRef(begin, end).withPrefix(tenantMapPrefix);
-
+Future<std::vector<std::pair<TenantName, TenantMapEntry>>> listTenantsTransaction(Transaction tr,
```

Comment:
Will this change in return type affect any of the handlers of this output? (e.g. fdbcli, python and java bindings)

Reply:
No, those all go through the special keys. The special keys wrappers of this function can use this function, and they present the data in the same way to the caller as they did before.